primefaces/src/main/resources/META-INF/resources/primefaces/picklist/picklist.js

```diff
@@ -351,6 +362,8 @@ PrimeFaces.widget.PickList = PrimeFaces.widget.BaseWidget.extend({
                 $this.focusedItem = list.children('.ui-picklist-item:visible:first');
             }
 
+            list.children('.ui-picklist-outline').removeClass('ui-picklist-outline');
```

Comment:
This is because I noticed a bug when selecting a new item by MOUSE it was leaving the old selection still outlined.